src/mlpack/methods/lmnn/lmnn_main.cpp

```diff
@@ -89,7 +89,9 @@ PROGRAM_INFO("Large Margin Nearest Neighbors (LMNN)",
     PRINT_PARAM_STRING("batch_size") + "), and the maximum number of passes "
     "(specified with " + PRINT_PARAM_STRING("passes") + ").  In "
     "addition, a normalized starting point can be used by specifying the " +
-    PRINT_PARAM_STRING("normalize") + " parameter. "
+    PRINT_PARAM_STRING("normalize") + " parameter. Furthermore, " +
+    "mean-centering can be performed on the dataset by specifying the " +
+    PRINT_PARAM_STRING("center") + "parameter. "
```

Comment:
Actually I realized that there's one more thing, which is that if we learn a metric using a mean-centered dataset, we can't then apply that metric to the non-mean-centered dataset.  So we should also add an output parameter `centered_data` or something like this, so that a user can have access to their mean-centered data for later use.  What do you think?

Reply:
Fair enough!